### DISCUSSION TITLE
cli tool for framework on server

### DISCUSSION THREAD

**Discussion by jjansenvr** [frankframework/frankframework#7680] [Ideas]
### Related Problem or Business Case

Due to the fact we are now running on a environment without an application server the information provided is no longer available (could be in the console)

### Describe the Feature

Not sure if it is a good idea but could we create cli for specific tasks from a server perspective 
For example
- getting jmx logging
- tranaction manager info
- information about the drivers (db , queue topics) 
-  reload configs

### Collaboration with External Companies

No, community resolution only

**Comment by nielsm5** [comment]
What is this about? We have plans to get rid of JMX due to nobody using this. Information about drivers/queues ed is visible via the console? What exactly is not available?

**Reply by jjansenvr** [reply]
It is not about what is not available through the console but extra functionality to use the f.e. in CICD pipelines or by "system administrators" to see info and start some actions on the frank command line

**could be** 
- trigger unit tests
- upload new config
- get the version of the FF
- etc

**Reply by Laurens-makel** [reply]
Reminds me of a proposal I once drafted, see my comment on #4453

**Reply by nielsm5** [reply]
I see there's a demand for this. I'll try and see if we can put this somewhere on our roadmap 😅 .
